resources/views/competitions/index.blade.php

```diff
@@ -3,5 +3,16 @@
 @section('main_content')
 
     <h1>All Competitions</h1>
+    <a href="/competitions/create"><h3>Create a new competition</h3></a>
```

Comment:
This would be good to use the `route()` helper for: `<a href="{{ route('competitions.create') }}">`